docs/source/scylla_cluster_crd.md

```diff
@@ -74,9 +74,29 @@ spec:
 ### Cluster Settings
```

Comment:
for some of these it will be useful to include an example, could also serve as disambiguation for fields like intensity which if I recall correctly need to be specified as a string due to lack of float support.

Reply:
There's an example at the top. I added comment about string value in intensity.